openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -163,13 +166,29 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 		etcdInitialClusterEntries = append(etcdInitialClusterEntries, fmt.Sprintf("%s=%s", member.Name, member.PeerURLs[0]))
 	}
 	if len(targetMember.Name) == 0 {
+		archiveDataDir(o.DataDir)
```

Comment:
You won't get here, right?   If the directory exists, then we returned way back at the top of the method.